Menu/Menu.js

```diff
@@ -1,10 +1,27 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+  menu.classList.toggle('menu--open');
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.getElementsByClassName('menu')[0];
+
+class menuitem
+{
+  constructor(element)
+  {
```

Comment:
It seems that you are not currently using this class (don't see any instance of it being used)? Also, to comply with naming convention, you'll want to type in your class names in PascalCase.